Sources/CouchDB/ConnectionProperties.swift

```diff
@@ -47,7 +45,7 @@ public struct ConnectionProperties {
     ///     - secured: Whether or not to use a secured connection.
     ///     - username: CouchDB admin username. Defaults to `nil`.
     ///     - password: CouchDB admin password. Defaults to `nil`.
-    public init(host: String, port: Int16, secured: Bool, username: String?=nil, password: String?=nil) {
+    public init(host: String, port: Int, secured: Bool, username: String?=nil, password: String?=nil) {
```

Comment:
Given port is later cast to Int16 could we update the doc here to indicate a maximum value, or perhaps even prevent it being set above the max value?